pkgs/servers/web-apps/healthchecks/default.nix

```diff
@@ -0,0 +1,86 @@
+{ lib
+, writeText
+, fetchFromGitHub
+, nixosTests
+, python3
+}:
+let
+  py = python3.override {
+    packageOverrides = self: super: {
+      django = super.django_4;
+      fido2 = let
+        version = "0.9.3";
+      in super.fido2.overridePythonAttrs(old: {
+        inherit version;
+        src = super.fetchPypi {
+          pname = "fido2";
+          inherit version;
```

Comment:
```suggestion
      fido2 = super.fido2.overridePythonAttrs(old: rec {
        version = "0.9.3";
        src = super.fetchPypi {
          pname = "fido2";
          inherit version;
```

Reply:
Ah, still not sure when rec is frowned upon and when not.

Reply:
Its fine to use rec here.